src/components/CarouselImage/styles.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../styles/shadows';
+@import '../../styles/colors';
+@import '../../styles/border-radius';
+
+.rainbow-carousel__panel {
+    -ms-flex: 0 0 100%;
+    flex: 0 0 100%;
+    max-width: 100%;
+}
+
+.rainbow-carousel__panel-action {
+    display: block;
+    border: 1px solid #dddbda;
+    border-radius: $border-radius-1;
+    overflow: hidden;
+}
+
+.rainbow-carousel__panel-action:focus {
+    outline: 0;
+    box-shadow: $shadow-outline;
+    border-color: $color-brand;
+}
+
+.rainbow-carousel__image {
+    border-top-left-radius: 0.25rem;
+    border-top-right-radius: 0.25rem;
+    overflow: hidden;
+
+    & > img {
```

Comment:
you don't need use this selector. 
Put a className to the image for apply to styles